Project.toml

```diff
@@ -15,6 +14,12 @@ SimpleTraits = "699a6c99-e7fa-54fc-8d76-47d257e15c1d"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
+[weakdeps]
+SharedArrays = "1a1011a3-84de-559e-8e89-a11a2f7dc383"
```

Comment:
Shouldn't SharedArrays then be removed from the [deps] section above as well?

Reply:
This was left behind for https://pkgdocs.julialang.org/v1/creating-packages/#Transition-from-normal-dependency-to-extension, but now that we're bumping compat to 1.10+, this can indeed be removed